Moderate-Background-Image/js/app.js

```diff
@@ -7,6 +7,7 @@ import {
 
 const bgUrl1 = "https://neru-b617e2b2-basic-dev.euw1.runtime.vonage.cloud/bg-1.png";
 const bgUrl2 = "https://neru-b617e2b2-basic-dev.euw1.runtime.vonage.cloud/bg-2.png";
+
```

Comment:
These are not found as of day. Happy to change them with the ones you guys share.